modules/integration/src/test/scala/scala/cli/integration/PublishTestDefinitions.scala

```diff
@@ -271,4 +271,68 @@ abstract class PublishTestDefinitions(val scalaVersionOpt: Option[String])
       expect(mainClasses == Set(scalaFile1, scalaFile2, s"$scriptsDir.${scriptName}_sc"))
     }
   }
+
+  test("missing secret key pasword") {
+    val baseExpectedArtifacts = Seq(
```

Comment:
NIT this could be a `Set` from the get-go, then no `.toSet` would be necessary below
```suggestion
    val baseExpectedArtifacts = Set(
```